components/add_groups_to_team_modal/add_groups_to_team_modal.test.tsx

```diff
@@ -154,28 +151,26 @@ describe('components/AddGroupsToTeamModal', () => {
     });
 
     test('should match when renderOption is called', () => {
-        const wrapper = shallow(
+        const wrapper = shallow<AddGroupsToTeamModal>(
             <AddGroupsToTeamModal {...baseProps}/>,
         );
 
-        const option = {id: 'id', last_picture_update: '12345', email: 'test@test.com'};
+        const option = {id: 'id_1', label: 'label_1', value: 'value_1'};
         let isSelected = false;
         function onAdd() {} //eslint-disable-line no-empty-function
+        function onMouseMove() {} //eslint-disable-line no-empty-function
```

Comment:
Could you refactor all empty-functions to this (or similar):

`const onMouseMove = jest.fn()`

That way we do not need to tell eslint to disable the rule and it is captured by jest (if we need it in a later iteration of the tests).